### PR TITLE
[eldbus] Change EAPI symbols for ELDBUS_API

### PR DESCRIPTION
**Pull request by Coquinho**

### CODE REVIEW

File: src/lib/eldbus/eldbus_api.h [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#  endif
+# else
+#  define ELDBUS_API
+# endif
+# define ELDBUS_API_WEAK
+#else
+# ifdef __GNUC__
+#  if __GNUC__ >= 4
+#   define ELDBUS_API __attribute__ ((visibility("default")))
+#   define ELDBUS_API_WEAK __attribute__ ((weak))
```
Comment (Coquinho) [inline src/lib/eldbus/eldbus_api.h:23]: weak don't seems to be used.

**Comment by Coquinho** [comment]
> I think we need to add `-DELDBUS_BUILD` to `src/lib/eldbus/meson.build:102`.

@joaoantoniocardoso Done.

File: src/lib/eldbus/eldbus_api.h (hunk 1 above)
Comment (joaoantoniocardoso) [reply src/lib/eldbus/eldbus_api.h:23]: It seems that we are defining _weak_ for all libs and not necessarily using them

Comment (Coquinho) [reply src/lib/eldbus/eldbus_api.h:23]: yeah, I don't know if people at phab will like that.
Should we just do it like this for now, and later update it with phabs?

Comment (joaoantoniocardoso) [reply src/lib/eldbus/eldbus_api.h:23]: Yeah I think. It will not be hard to remove definitions from all APIs if necessary

**Comment by Coquinho** [comment]
Hey @joaoantoniocardoso, @walac, there were changes after your approval, so please look again.

**Pull request merged**